docs/dev/development-process/development-process.rst

```diff
@@ -0,0 +1,203 @@
+       ``Adaptive learning``, ``Learning path``, ``Tutorial groups``, ``Iris``.
+    2. If the change is not visible to end users, or it is a pure development or test improvement, we use the term “``Development``:”.
+    3. Everything else belongs to the ``General`` category.
+
+2. The colon is not highlighted.
+
+3. After the colon, there should be a verbal form that is understandable by end users and non-technical persons, because this will automatically become part of the release notes.
+
+    1. The text should be short, non-capitalized (except the first word) and should include the most important keywords. Do not repeat the feature if it is possible.
+    2. We generally distinguish between bugfixes (the verb “Fix”) and improvements (all kinds of verbs) in the release notes. This should be immediately clear from the title.
+    3. Good examples:
+
+        - “Allow instructors to delete submissions in the participation detail view”
+        - “Fix an issue when clicking on the start exercise button”
+        - “Add the possibility for instructors to define submission policies”
+
+
+
+Steps to Create and Merge a Pull Request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**0. Precondition -> only Developer**
+
+* Limit yourself to one functionality per pull request.
+* Split up your task in multiple branches & pull requests if necessary.
+* `Commit Early, Commit Often, Perfect Later, Publish Once. <https://speakerdeck.com/lemiorhan/10-git-anti-patterns-you-should-be-aware-of>`_
+
+**1. Start Implementation -> only Developer**
+
+* `Open a draft pull request. <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_ This allows for code related questions and discussions.
+
+**2. Implementation is "done" -> only Developer**
+
+* Make sure all steps in the `Checklist <https://github.com/ls1intum/Artemis/blob/develop/.github/PULL_REQUEST_TEMPLATE.md>`_ are completed.
+* Add or update the "Steps for Testing" in the description of your pull request.
+* Make sure that the changes in the pull request are only the ones necessary.
+* Mark the pull request as `ready for review. <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request>`_
+
+**3. Review**
+
+Developer:
+
+* Organize or join a testing session. Especially for large pull requests this makes testing a lot easier.
+* Actively look for reviews. Do not just open the pull request and wait.
+
+Reviewer:
+
+* Perform the "Steps for Testing" and verify that the new functionality is working as expected.
+* Verify that related functionality is still working as expected.
+* Check the changes to
+    * conform with the code style.
+    * make sure you can easily understand the code.
+    * make sure that (extensive) comments are present where deemed necessary.
+    * performance is reasonable (e.g. number of database queries or HTTP calls).
```

Comment:
> * Check the changes to
    * conform with the code style.
    * make sure you can easily understand the code.
    * make sure that (extensive) comments are present where deemed necessary.
    * performance is reasonable (e.g. number of database queries or HTTP calls).

The wording doesn't really work here. I would suggest:
 * Ensure that the code changes...
    * conform to the code style guide.
    * are easily understandable.
    * contain thorough documentation where applicable.
    * maintain reasonable performance (e.g. no excessive/unnecessary database queries or HTTP calls).